File: src/main/java/yeh/pratice/leetcode/TwoSumRun.java
package yeh.pratice.leetcode;

public class TwoSumRun {
    public static void main(String[] args) {
        int [] nums={1,8,5,7};
        int target=9;
        TwoSum twoSum=new TwoSum();
        int []test=twoSum.Target(nums,target);
        for(int i=0;i<test.length;i++){
            System.out.println(test[i]);
        }

    }
}

File: src/main/kotlin/yeh/pratice/leetcode/TwoSumRun.kt
package yeh.pratice.leetcode.kotlin

fun main() {
    val nums= intArrayOf(1,3,5,7)
    val target=8
    val twosum= TwoSum()
    val test=twosum.getTwoSum(nums,target)
    var i=0
    while (i<test.size){
        println(test[i])
        i++
    }

}
class TwoSum(){

    fun getTwoSum(nums: IntArray,target: Int):IntArray{
        var answer=IntArray(2)

        var i=0

        while(i<nums.size){
            var j=i+1
            while (j<nums.size){
                if(nums[i]+nums[j]==target)
                {answer=intArrayOf(i,j)}
                j++
            }
            i++
        }
        return answer
    }
}